code/modules/surgery/tools.dm

```diff
@@ -235,3 +235,28 @@
 	else
 		to_chat(user, "[src] is empty.")
 	return
+
+/obj/item/surgical_processor //allows medical cyborgs to scan and initiate advanced surgeries
+	name = "Surgical Processor"
+	desc = "A device for scanning and initiating surgeries from a disk or operating computer."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "spectrometer"
+	var/list/advanced_surgeries = list()
+	
+/obj/item/surgical_processor/afterattack(obj/item/O, mob/user, proximity)

```

Comment:
Scanning the surgical computer with the module still slaps it (attack animation, sound).

Reply:
Would changing the proc from afterattack() to attack() be an acceptable way to fix this?

Reply:
nevermind, forgot about itemflags